Library/Formula/gcovr.rb

```diff
@@ -0,0 +1,17 @@
+class Gcovr < Formula
+  homepage "http://gcovr.com/"
+  url "https://github.com/gcovr/gcovr/archive/3.2.tar.gz"
+  sha1 "97e44794969378db99cb77ee3c71b110dc634a8f"
+  head "https://github.com/gcovr/gcovr.git"
+
+  def install
+    prefix.install Dir["*"]
+    bin.install_symlink "../scripts/gcovr"
+  end
+
+  test do
+    system "#{bin}/gcovr", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!